templates/prelogin/demo.html

```diff
@@ -27,19 +27,45 @@
     <script>
         {% include 'app_manager/partials/form_designer_requirejs_config.html' with vellum_debug=None %}
 
-        require(["options", "jquery", "jquery.vellum"], function (options, $) {
+        require(["jquery", "jquery.vellum"], function ($) {
             $(function () {
-                var VELLUM_OPTIONS = options.options;
-                VELLUM_OPTIONS.plugins = ["modeliteration", "itemset", "atwho"];
-                VELLUM_OPTIONS.core.saveType = 'full';
-                VELLUM_OPTIONS.core.saveUrl = function(data) {
-                    $("#signupModal").modal();
-                };
-                VELLUM_OPTIONS.windowManager = {
-                    leftOffset: 0,
-                    topOffset: 0,
-                };
-                $('#formdesigner').vellum(VELLUM_OPTIONS);
+                $('#formdesigner').vellum({
+                    core: {
+                        loadDelay: 0,
```

Comment:
Pretty sure this should only be set for automated tests. Can't remember what this is used for, but I think it's related to the spinner visibility or something like that. Should be safe to remove this line, causing vellum to use the default value.